server/src/model/user.model.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require("mongoose");
+
+const userSchema = mongoose.Schema({
+  email: {
+    type: String,
+    required: true,
+    unique: true,
+    match: /[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?/
```

Comment:
Source? I'm curious

Reply:
Well, 'unique' and 'match' will also be removed when the sign up is removed

This is the source, not where I got it from ;)
https://mongoosejs.com/docs/schematypes.html#string-validators